### DISCUSSION TITLE
Types of assistants or trainees as a way of introduction to a complex and important profession

### DISCUSSION THREAD

**Discussion by NovikovPavel** [unitystation/unitystation#6988] [Ideas]
I have seen many times how "professional" medical, engineers and others make incredible mistakes, when they should be good professionals, without which the normal operation of the station is impossible. Or they just say that the newcomers are here and do not know how to do anything. This is because they don't have experience, and they won't have it if they don't go there. But they need to go with experience and in the end they get a vicious circle.

For example: an engineer who can't start an engine or just turn on the solar panels.

So, I suggest adding a feature already created in the original game - types of assistants.

Trainee engineer.
A trainee engineer learns how the station's systems work and how to maintain them. It also examines rules and procedures for emergency situations.

Intern.
An intern learns how to perform surgery and patient care or training with a virologist.

Junior researcher.
A junior researcher learns to work in the chosen research area.

Security cadet.
A security cadet learns to work with criminals. In most cases, in terms of studying their psychology and behavior.

Of course, assistants must have basic access to their compartments.

**Comment by corp-0** [comment]
I like the idea of learning roles a lot. First saw it in Aurora station and it really can be not just helpful, but also fun.

Since the project has improving the new player experience as an objective, this suggestion is a perfect match.
